### DISCUSSION TITLE
PB bot: Diferencias entre primera versión con la 2.0.0

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#235] [2. Questions and Answers] [answered]
Pego el texto 

Creo que hay un bug en los bot y rutinas del surveytrigger, no se si es feo o no , quizás los detectaron antes. Si en mi bot o rutina del survey trigger agrego otro pb bot, éste me quedará como versión 2.0.0, mientras que los otros bots estarán en la primera versión. cuando lo ejecuto se me ejecuta bien, salvo el bot que tiene la versión 2.0.0. Sin embargo, si cambio la versión a primera versión, aparte de que se me borra todo lo registrado en ese bot, y hago nuevamente un qa, éste se me ejecuta.

Favor su ayuda o respuesta :>

gracias

**Comment by guillevalin** [comment]
@NicolasPLG efectivamente la aplicación te sugerirá actualizar los bots a la nueva versión de las etapas existentes, pero las nuevas etapas partirán en la nueva versión.

Cuál es tu problema en específico? Creo que no te entendí bien. No debería haber problema en mezclar versiones siempre y cuando respetes los inputs/outputs de cada bot, en general los 2.0.0 recibieron mejoras respecto a la claridad de los parámetros de entrada y salida y mejoras en el manejo de errores.

**Reply by NicolasPLG** [reply]
hola @guillevalin ya lo hablé con @eduborgono hace un ratito atrás y me explicó las diferencias que habian.

gracias :D

**Comment by eduborgono** [comment, marked as answer]
@NicolasPLG lo que sucedió fue que la stage de la rutina que está en la versión 2.0.0 falló por alguna razón. Se puede ver el detalle del error dentro del registro del bot, en el apartado de resultados y la etapa que está dando problemas como en el siguiente ejemplo

![image](https://user-images.githubusercontent.com/22122681/120826614-a571e280-c528-11eb-9e9c-55518b87706d.png)
